main/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     implementation 'com.google.guava:guava:29.0-jre'
     implementation 'com.google.flogger:flogger:0.5.1'
     implementation 'com.google.flogger:flogger-system-backend:0.5.1'
+    implementation 'io.sgr:s2-geometry-library-java:1.0.1'
```

Comment:
Nice! I didn't know that this library existed.

Reply:
I am glad to discover it! That is the Google default library for Earth geometry.

Reply:
I am glad to discover it :) This is Google default library for geometry.

Reply:
@aababilov Do you know if Google plans to update the Java port (https://github.com/google/s2-geometry-library-java) soon? Looks like the Java port was last updated in 2011 (😱), while the main [C++ version](https://github.com/google/s2geometry) was updated about two weeks ago (active development).